pkg/k8shandler/cluster.go

```diff
@@ -200,17 +201,22 @@ func (er *ElasticsearchRequest) getNodeUpgradeInProgress() NodeTypeInterface {
 
 func (er *ElasticsearchRequest) progressUnschedulableNodes() error {
 	cluster := er.cluster
+	clusterNodes := nodes[nodeMapKey(cluster.GetName(), cluster.GetNamespace())]
+
+	for _, nodeStatus := range cluster.Status.Nodes {
+		if isPodUnschedulableConditionTrue(nodeStatus.Conditions) ||
+			isPodImagePullBackOff(nodeStatus.Conditions) ||
+			isPodCrashLoopBackOff(nodeStatus.Conditions) {
+
+			for _, node := range clusterNodes {
+				if nodeStatus.DeploymentName == node.name() || nodeStatus.StatefulSetName == node.name() {
+					if node.isMissing() {
+						log.Info("Unschedulable node not existing, skip progressing node changes", "node", node.name())
```

Comment:
nit but maybe reword to be something like "Unschedulable node does not have a k8s object, skipping"  ?

Reply:
sure!